web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
```

Comment:
For a simple object of data, I think that's too much. If this was a service of methods that did meaningful things (like massage data or change stuff around), or there was a requirement to privatise `settings` so it cannot be accessed directly or changed, then 👍 , but I don't think that's the case